### ISSUE TITLE
Fortran computational error (1e-6) does not support bounds condition checking in Python

### ISSUE THREAD

**Issue by nghi-truyen** [DassHydro-dev/smash#23]
The bound check in Python standardize:
```python
# % Check that parameters and states are inside user bounds
        for i, name in enumerate(control_vector):
            if name in setup._parameters_name:
                ind = np.argwhere(setup._parameters_name == name)

                parameters_attr = getattr(parameters, name)
                if np.any(parameters_attr + 1e-6 < bounds[i, 0]) or np.any(
                    parameters_attr - 1e-6 > bounds[i, 1]
                ):
                    raise ValueError(
                        f"bounds values for '{name}' are invalid, background parameters [{np.min(parameters_attr)} {np.max(parameters_attr)}] is outside the bounds {bounds[i,:]}"
                    )

            # % Already check, must be states if not parameters
            else:
                ind = np.argwhere(setup._states_name == name)

                states_attr = getattr(states, name)
                if np.any(states_attr + 1e-6 < bounds[i, 0]) or np.any(
                    states_attr - 1e-6 > bounds[i, 1]
                ):
                    raise ValueError(
                        f"bounds values for '{name}' are invalid, background states [{np.min(states_attr)} {np.max(states_attr)}] is outside the bounds {bounds[i,:]}"
                    )
``` 
which does not support for a tolerance of 1e-6 in some cases:
```bash
ValueError: bounds values for 'cft' are invalid, background parameters [500.0 1000.0001220703125] is outside the bounds [   1. 1000.]
```

**Comment by nghi-truyen** [comment]
@inoelloc diminue à 1e-3 ou comment qu'est-ce que tu proposes?

**Comment by inoelloc** [comment]
Issue resolved in PR #24
